tests/test_server_proto.py

```diff
@@ -517,6 +517,12 @@ async def test_server_proto_set_global_01(self):
 
         await self.con.query('reset global glob')
 
+    async def test_server_proto_set_global_02(self):
+        await self.con.execute('START TRANSACTION')
+        await self.con.execute('set global glob := 1')
```

Comment:
This makes me wonder if we want to support transaction-scoped globals.

Reply:
I think that in practice with the new protocol, this won't matter at all?

Reply:
Since setting globals will be done on a language binding level and not via `SET GLOBAL`

Reply:
Sure, we can circle back to this when and if this becomes needed at the language level.